src/Microsoft.AspNet.Http/DefaultHttpContext.cs

```diff
@@ -168,8 +185,26 @@ public override void Abort()
 
         public override void Dispose()
         {
-            // REVIEW: is this necessary? is the environment "owned" by the context?
```

Comment:
Question has moved to -> https://github.com/aspnet/HttpAbstractions/pull/404/files#diff-ad404d59bc5860d71b7403b8e0a665b2R102